packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -579,7 +611,7 @@ export function bailoutHooks(
   current.lanes = removeLanes(current.lanes, lanes);
 }
 
-export function resetHooksAfterThrow(): void {
+export function resetHooksAfterThrow(erroredWork: Fiber | null): void {
```

Comment:
Isn't `erroredWork` the same as `currentlyRenderingFiber` at this point which is also used elsewhere in this reset function. for consistency maybe just use the module global?

Reply:
I confirmed that `currentlyRenderingFiber` can be null here. @acdlite is there some global for "current fiber" that I can rely on here? Or is the approach of passing in the fiber from the caller ok?